integrationtests.py

```diff
@@ -151,6 +151,16 @@ def testRecompileObjectSetOtherDir(self):
         subprocess.check_call(cmd) # Compile once
         subprocess.check_call(cmd) # Compile again
 
+    def testPipedOutput(self):
+        with cd(ASSETS_DIR):
+            cmd = [PYTHON_BINARY, CLCACHE_SCRIPT, "/nologo", "/c", "fibonacci.c", "/Fofibonacci_c.obj"]
```

Comment:
Do we really know that there are some proper line endings from this call? As far as I know, there is only the file name.

What about adding dropping /nologo?